tests/test_morphsqueeze.py

```diff
@@ -0,0 +1,74 @@
+"""
+The squeeze morph is used to correct for small non-linear geometric distortions
+from detectors that are not effectively corrected during calibration, such as
+individual module misalignment or tilt. This squeezing is applied as:
+x_squeezed = x + squeeze_0 + squeeze_1 * x**2 + squeeze_2 * x**3
+
+The squeeze distortions that we might encounter practically are going to be
+very small. Furthermore, large values for the squeezing parameters lead to
+missing values during interpolation. Therefore is important to use small
+squeezing values to avoid error or unphysical results.
+Safe bounds for the squeezing parameters are:
+squeeze_0 [-0.2, 0.2]
+squeeze_1 [-0.001, 0.001]
+squeeze_2 [-0.0001, 0.0001]
+Values outside these bounds should be used carefully.
+Note that these bounds are established for an x-axis that goes from 0 to 10.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.interpolate import interp1d
+
+
```

Comment:
we are now going to test 3 or 4 different variants of the same thing (i.e., different representative numbers) so it is time to use @pytest.paramatrize decorator.  Please see `diffpy.utils` tests for examples.